backend/PythonClient/multirotor/airsim_application.py

```diff
@@ -105,3 +105,39 @@ def get_cesium_origin(self):
         lon = data["longitude"]
         height = data["height"]
         return [lat, lon, height]
+    
+@app.route('/uploadMission', methods=['POST'])
+def upload_file():
+    log_text = "" 
+    airsim_app = AirSimApplication()  # Create an instance of AirSimApplication
+
+    try:
+      file = request.files['file']
+      filename = file.filename
+      if not filename: 
+          raise ValueError("No file Provided");
+
+      custom_mission_dir = '../multirotor/mission/custom'
+      path = os.path.join(custom_mission_dir, filename)
+      file.save(path)
+      
+      #Code for Log success
+
+      airsim_app.append_pass_to_log(f"Report '{filename}' successfully uploaded at {path}.")
```

Comment:
Files don't seem to be appearing for the acceptance report feature. Might need to rework this part